### PR TITLE
new app-id for  NewsFlash

### PR DESCRIPTION
**Pull request by jangernert**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] ~~Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*~~

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### Re-publish of NewsFlash

This is an attempt to re-publish NewsFlash with a new app-id that matches the gitlab group url in order to get the app verified.

- the app id changed from `com.gitlab.newsflash` -> `com.gitlab.news_flash`
- the group url changed from `https://gitlab.com/news-flash` to `https://gitlab.com/news_flash`

Discourse thread asking about the best way to get NewsFlash verified: https://discourse.flathub.org/t/manual-verification/4780/2

### CODE REVIEW

**Comment by jangernert** [comment]
Ah, I see. I think I remember now my "legacy" app-id being on a whitelist somewhere. I'm going to update it later today.

File: com.gitlab.news_flash.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+    "app-id" : "com.gitlab.news_flash",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command" : "com.gitlab.newsflash",
```
Comment (hfiguiere) [inline com.gitlab.news_flash.json:9]: hasn't this changed too?

File: com.gitlab.news_flash.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--device=dri",
+        "--talk-name=org.freedesktop.Notifications"
```
Comment (hfiguiere) [inline com.gitlab.news_flash.json:17]: do you really need this D-Bus name? Gtk handle notifications properly without it.

Comment (jangernert) [reply com.gitlab.news_flash.json]: I didn't know that. Thanks!

**Comment by jangernert** [comment]
bot, build io.gitlab.news_flash.NewsFlash

**Comment by jangernert** [comment]
bot, build io.gitlab.news_flash.NewsFlash

**Pull request closed without merging**